src/components/CheckIn/ItemsListModal.js

```diff
@@ -49,6 +61,15 @@ const ItemsListModal = ({
         marginBottom0
       />
     ),
+    'callNumber': (item) => (
+      <TextField
+        disabled={isLoading || item.pieceFormat !== 'Physical'}
+        onChange={(e) => onChangeField(item, e.target.value, 'callNumber')}
```

Comment:
maybe it's better to use `useCallback` for this

Reply:
I think there is no way to do this here as it depends on item

Reply:
if you look into `onChangeField = (item, value, key) => {` there is no `item` usage - only item.id. And this also could be passed through MCL - `this.props.onRowClick(e, row);`

Reply:
due to next tasks for separate receiving app maybe it's better to provide as less changes as possible